talks/_posts/2019-08-30-welcome-to-ritlug.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Fall 2019 - Welcome to RITlug!
+author: Jeffery Russell
+layout: post
+slides: 2019-w01-fall-into-to-linux.pdf
+date: 2019-08-30
+---
+
+Welcome to another year of the RIT Linux Users Group, fall 2016 edition!
```

Comment:
What do you mean by **2016** edition?

Reply:
dope, I think I just copied in the lingo from the 2016 intro slide post